### ISSUE TITLE
Selecting a Collection from a database of mongoDB

### ISSUE THREAD

**Issue by alexandrefabre22** [compose/transporter#425]
Hello,

I tested transporter on a MongoDB and I selected a database named db.

var source = mongodb ({
"uri": "mongodb://10.0.130.97:27017/db"
})

db has three Collections: doc, fs.files, fs.chunks. 
I would like just to send “doc” to RabbitMQ with transporter, without fs.files and fs.chunks.

Would you have a solution for this case?

Thanks!

**Comment by jipperinbham** [comment]
yes, this is where the namespace filters come in. In the pipeline definition you'd have the following:

```javascript
var source = mongodb ({
"uri": "mongodb://10.0.130.97:27017/db"
})

var rabbitmq = rabbitmq({
"uri": "amqps://[user]:[password]@HOST:PORT/VHOST"
})
t.Source("mongodb", source, "/^doc$/").Save("rmq", rabbitmq)
```

Be sure to take a look at the [rabbitmq adaptor config](https://github.com/compose/transporter/blob/master/adaptor/rabbitmq/README.md) for all the options.
